### PR TITLE
Feature: SOS Portal 

### PR DESCRIPTION
**Pull request by rasmusguterstam**
# Jira Issue: [IP-216](https://hedvig.atlassian.net/browse/IP-216)

## What?
- Add portal with limited insurance information for SOS International

## Optional screenshots

https://user-images.githubusercontent.com/51323202/146797280-90ee1a58-d77b-4a45-9b45-6310672a9afe.mov

### CODE REVIEW

File: src/portals/sos/pages/MainPage.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import chroma from 'chroma-js'
+import { useAuthenticate } from 'portals/hope/features/user/hooks/use-authenticate'
+import { Route, Switch } from 'react-router'
+
+const Container = styled.div`
+  width: 100vw;
+  min-height: 90vh;
+  padding-left: 10vw;
+  padding-right: 10vw;
+  padding-bottom: 5rem;
```
Comment (maxplt) [inline src/portals/sos/pages/MainPage.tsx:14]: Do these paddings really make sense? 🧐

File: src/portals/sos/pages/MainPage.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import React from 'react'
+import styled from '@emotion/styled'
+import { MemberSearchForm } from 'portals/sos/features/member-search/MemberSearchForm'
+import chroma from 'chroma-js'
+import { useAuthenticate } from 'portals/hope/features/user/hooks/use-authenticate'
+import { Route, Switch } from 'react-router'
+
+const Container = styled.div`
+  width: 100vw;
+  min-height: 90vh;
```
Comment (maxplt) [inline src/portals/sos/pages/MainPage.tsx:11]: What if instead create `Container` with `Content` and `Footer` inside it?
And make `Container` with `width: 100vw`, `height: 100vh`, `display: flex` and `flex-direction: column`
And then just set to `Content` element `flex: 1`. 
In my opinion, it'll be more clear

File: src/portals/sos/pages/MainPage.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            redirectToLogin()
+            return null
+          }}
+        />
+      </Switch>
+    )
+  }
+
+  return (
+    <>
```
Comment (maxplt) [inline src/portals/sos/pages/MainPage.tsx:68]: `Container`?

File: src/portals/sos/pages/MainPage.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      <Container>
```
Comment (maxplt) [inline src/portals/sos/pages/MainPage.tsx:69]: `Content`?

**Review by maxplt — APPROVED** [review]
Looks really good! 🤩
Just added some minor thoughts about MainPage styles

File: src/portals/sos/pages/MainPage.tsx (hunk 2 above)
Comment (rasmusguterstam) [reply src/portals/sos/pages/MainPage.tsx:11]: Really good point! 👍

File: src/portals/sos/pages/MainPage.tsx (hunk 1 above)
Comment (rasmusguterstam) [reply src/portals/sos/pages/MainPage.tsx:14]: Hehe no not really anymore 🤓

**Pull request merged**